packages/db/src/search/GranuleSearch.ts

```diff
@@ -116,9 +119,30 @@ export class GranuleSearch extends BaseSearch {
    * @param pgRecords - postgres records returned from query
    * @returns translated api records
    */
-  protected translatePostgresRecordsToApiRecords(pgRecords: GranuleRecord[])
-    : Partial<ApiGranuleRecord>[] {
+  protected async translatePostgresRecordsToApiRecords(pgRecords: GranuleRecord[], knex: Knex)
+    : Promise<Partial<ApiGranuleRecord>[]> {
     log.debug(`translatePostgresRecordsToApiRecords number of records ${pgRecords.length} `);
+
+    const cumulusIds = pgRecords.map((record) => record.cumulus_id);
+
+    //get Files
+    const fileModel = new FilePgModel();
+    const files = await fileModel.searchByGranuleCumulusIds(knex, cumulusIds);
+    const fileMapping: { [key: number]: PostgresFileRecord[] } = {};
+    files.forEach((file) => {
+      if (!(file.granule_cumulus_id in fileMapping)) {
+        fileMapping[file.granule_cumulus_id] = [];
+      }
+      fileMapping[file.granule_cumulus_id].push(file);
```

Comment:
Looked into this a moment -and this doesn't need a translation as the non-db granule translator handles it.   👍🏻  

Nit: It might be worth making sure there's an explicit test covering files getting translated at this method level, with the caveat I haven't looked at test coverage yet.

Reply:
this translation is now explicitly tested